packages/plugin-user/src/config.js

```diff
@@ -8,6 +8,11 @@ export default {
   user: {
     batcherWait: 100,
     batcherMaxCalls: 100,
-    batcherMaxWait: 1500
+    batcherMaxWait: 1500,
+    registrationDefaults: {
```

Comment:
yea, i think it's still needed. the register endpoint was defined and finalized before contact with real-world usage, so it's got some weird requirements that only apply on mobile.